src/ensembl/src/shared/components/image-button/ImageButton.scss

```diff
@@ -41,28 +44,9 @@
 }
 
 .disabled {
-  cursor: not-allowed;
+  cursor: not-allowed !important;
```

Comment:
No way! `important`s are a swear word in modern CSS. Why would we need it here?

Besides, haven't we decided that the disabled class simply has `cursor: default`?